src/AbstractCsv.php

```diff
@@ -486,7 +486,7 @@ public function setOutputBOM(string $str): self
     public function addStreamFilter(string $filtername, $params = null): self
     {
         if (!$this->document instanceof Stream) {
-            throw new Exception('The stream filter API can not be used');
+            throw new CannotAddStreamFilter('The stream filter API can not be used');
```

Comment:
I'm wondering if we should not name this one `MissingStreamFilterSupport` or `StreamFilterSupportMissing`

Reply:
`StreamFilterSupportMissing` sounds good to me. Want me to change that?

Reply:
yes please